### PR TITLE
Migrated livetvstatus.js to es6 module

### PR DESCRIPTION
**Pull request by matjaz321**
**Changes**
Migration of `livetvstatus.js` to es6 module
**Issues**
Part of #868

### CODE REVIEW

File: src/controllers/livetvstatus.js [hunk 1]
```diff
@@ -1,294 +1,307 @@
...
-        // TODO move card creation code to Card component
-
-        if (layoutManager.tv) {
-            cssClass += ' show-focus';
-
-            if (enableFocusTransform) {
-                cssClass += ' show-animation';
-            }
+'use strict';
+
```
Comment (dkanada) [inline src/controllers/livetvstatus.js:2]: We want to remove this as well.

Comment (matjaz321) [reply src/controllers/livetvstatus.js:2]: 'use strict` why?

Comment (Camc314) [reply src/controllers/livetvstatus.js:2]: ES6 modules are automatically in `strict` mode see [here](https://www.ecma-international.org/ecma-262/6.0/#sec-strict-mode-code)

Comment (matjaz321) [reply src/controllers/livetvstatus.js]: Fixed

File: src/controllers/livetvstatus.js [hunk 2]
```diff
@@ -1,294 +1,305 @@
...
-                switch (id) {
-                    case 'delete':
-                        deleteDevice(dom.parentWithClass(button, 'page'), tunerDeviceId);
-                        break;
-
-                    case 'edit':
-                        Dashboard.navigate('livetvtuner.html?id=' + tunerDeviceId);
-                }
-            });
+    require(['actionsheet'], function (actionsheet) {
```
Comment (Camc314) [inline src/controllers/livetvstatus.js:268]:
```suggestion
    import('actionsheet').then(({default: actionsheet}) => {
```

Comment (matjaz321) [reply src/controllers/livetvstatus.js]: Ah nice one I missed it somehow

**Pull request merged**